Backend/routes/userRoutes.js

```
@@ -1,103 +1,116 @@
-      return res.status(404).json({ message: "User not found" });
+        if (!uid) {
+            return res.status(400).json({ message: "UID is required" });
+        }
+        const user = await User.findOne({ uid: uid });
+        if (!user) {
+            return res.status(404).json({ message: "User not found" });
+        }
+        res.status(200).json({ user });
+    } catch (error) {
+        console.error("❌ Error fetching user data:", error);
+        res.status(500).json({ message: "Internal server error: " + error.message });
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)